src/meta/src/stream/stream_manager.rs

```diff
@@ -1026,6 +1019,19 @@ mod tests {
                 .await?;
             cluster_manager.activate_worker_node(host).await?;
 
+            let fake_service = FakeStreamService {
+                worker_id: worker_node.id,
+                inner: state.clone(),
+            };
+            let stream_srv = StreamServiceServer::new(fake_service);
+            let join_handle = tokio::spawn(async move {
+                tonic::transport::Server::builder()
+                    .add_service(stream_srv)
+                    .serve_with_shutdown(addr, async move { shutdown_rx.await.unwrap() })
+                    .await
+                    .unwrap();
+            });
+
```

Comment:
What's this refactor for?

Reply:
After this PR, the `FakeStreamService` should take the `worker_node.id` before spawned. Therefore, we changed to create and spawn it after `add_worker_node` to get the `worker_node.id`.